packages/databricks-vscode/DATABRICKS.quickstart.md

```diff
@@ -1,7 +1,3 @@
-# ⚠️ Note
-
-> **The quickstart guide for `2.*.*` version of the extension is still work-in-progress. For now, please use the documentation we provided to you. Please reach out to your Databricks representative if you don't have access to the relevant docs.**
-
 # Databricks Extension for Visual Studio Code
 
 The Databricks extension for VS Code allows you to develop for the Databricks Lakehouse platform from VS Code.
```

Comment:
```suggestion
The Databricks extension for Visual Studio Code allows you to develop for the Databricks Data Intelligence Platform from Visual Studio Code.
```
This should really match the first line in the docs, which is currently "The Databricks extension for Visual Studio Code enables you to connect to your remote Databricks workspaces from Visual Studio Code." But it could be I should change the docs instead. Thoughts?